### ISSUE TITLE
Plugin incompatible with clion 2019.x

### ISSUE THREAD

**Issue by robert-alfaro** [daniel-sullivan/clion-embedded-esp32#1]
## Steps to Reproduce the Problem

  1.
  1.
  1.

## Specifications

  - Plugin Version:
  - CLion version:
  - OpenOCD version:
  - GDB Version:
  - STM32CubeMX version:
  - Toolchain:
  - Platform:
  - Hardware configuration:

**Comment by NoeelMoeskops** [comment]
changing `META_INF/plugin.xml` `<idea-version />` to `until-build="192.*"` in the `.jar` allows the plugin to be installed.
But when pressing the play button after configuration (according to `USAGE.md`, but I'm on Linux so who knows) it gives a exception:
```
java.lang.NoClassDefFoundError: com/jetbrains/cidr/cpp/execution/debugger/backend/GDBDriverConfiguration
	at esp32.embedded.clion.openocd.OpenOcdConfiguration.getState(OpenOcdConfiguration.java:101)
	at esp32.embedded.clion.openocd.OpenOcdConfiguration.getState(OpenOcdConfiguration.java:22)
	at com.intellij.execution.runners.ExecutionEnvironment.getState(ExecutionEnvironment.java:158)
	at com.intellij.execution.runners.BaseProgramRunner.execute(BaseProgramRunner.java:40)
	at com.intellij.execution.runners.BaseProgramRunner.execute(BaseProgramRunner.java:35)
	at com.intellij.execution.ProgramRunnerUtil.executeConfigurationAsync(ProgramRunnerUtil.java:92)
	at com.intellij.execution.ProgramRunnerUtil.executeConfiguration(ProgramRunnerUtil.java:40)
	at com.intellij.execution.impl.ExecutionManagerImpl.restart(ExecutionManagerImpl.java:93)
	at com.intellij.execution.impl.ExecutionManagerImpl.access$300(ExecutionManagerImpl.java:44)
	at com.intellij.execution.impl.ExecutionManagerImpl$3.run(ExecutionManagerImpl.java:465)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:232)
	at com.intellij.util.Alarm$Request.runSafely(Alarm.java:368)
	at com.intellij.util.Alarm$Request.run(Alarm.java:355)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:224)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:315)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:435)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:419)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:403)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:729)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:678)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:373)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: com.jetbrains.cidr.cpp.execution.debugger.backend.GDBDriverConfiguration PluginClassLoader[esp32.embedded.clion.armsupport.prj, 0.1.alpha1] com.intellij.ide.plugins.cl.PluginClassLoader@7509599c
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:51)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 37 more
```

**Comment by NoeelMoeskops** [comment]
I fixed it! see #2. You can download it in the meantime manually [on my repo](https://github.com/NoeelMoeskops/clion-embedded-esp32/releases)

**Comment by daniel-sullivan** [comment]
@NoeelMoeskops Thank you very much for that! I've been busy with non-embedded programming projects at work lately so hadn't been able to get around to fixing this. Much appreciated!

(New version also uploaded to the JetBrains Repo - should be up within two business days)
